.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+{
+  "version": "2.0.0",
```

Comment:
what does this do?

Reply:
Files for better developer experience. If you develop this extension in vscode you can easily start nom watch with a shortcut. I can add some info to README the same way I have in vscode-php-debug.